nxproxy/man/Makefile.am

```diff
@@ -0,0 +1,9 @@
+NULL =
+
+man_MANS =						\
```

Comment:
Changing that to `dist_man_MANS` should better reflect what you are trying to do - and the `EXTRA_DIST` definition can be also dropped when using `dist_man_MANS`.